lib/Backend/GlobOptFields.cpp

```diff
@@ -2170,7 +2170,7 @@ GlobOpt::FinishOptPropOp(IR::Instr *instr, IR::PropertySymOpnd *opnd, BasicBlock
         isObjTypeSpecialized = ProcessPropOpInTypeCheckSeq<true>(instr, opnd, block, updateExistingValue, emitsTypeCheckOut, changesTypeValueOut, &isObjTypeChecked);
     }
 
-    if (opnd == instr->GetDst() && this->objectTypeSyms && !isObjTypeChecked)
+    if (opnd == instr->GetDst() && this->objectTypeSyms)
```

Comment:
That's pretty funny. I obviously have vacation brain. I'll look again later.

-- Paul
________________________________
From: Akrosh Gandhi<mailto:notifications@github.com>
Sent: ‎9/‎2/‎2016 10:04 AM
To: Microsoft/ChakraCore<mailto:ChakraCore@noreply.github.com>
Cc: Paul Leathers<mailto:pleath@microsoft.com>; Mention<mailto:mention@noreply.github.com>
Subject: Re: [Microsoft/ChakraCore] Address deref issue (#1530)


In lib/Backend/GlobOptFields.cpp<https://github.com/Microsoft/ChakraCore/pull/1530#discussion_r77379371>:

> @@ -2170,7 +2170,7 @@ GlobOpt::FinishOptPropOp(IR::Instr *instr, IR::PropertySymOpnd *opnd, BasicBlock
>          isObjTypeSpecialized = ProcessPropOpInTypeCheckSeq<true>(instr, opnd, block, updateExistingValue, emitsTypeCheckOut, changesTypeValueOut, &isObjTypeChecked);
>      }
>
> -    if (opnd == instr->GetDst() && this->objectTypeSyms && !isObjTypeChecked)
> +    if (opnd == instr->GetDst() && this->objectTypeSyms)


Wait, I am porting the change you have made. Was there a problem in the porting?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/Microsoft/ChakraCore/pull/1530/files/5ec2d8f6dd3e67e8aa85002dbad152a614f92eeb#r77379371>, or mute the thread<https://github.com/notifications/unsubscribe-auth/APF8RCyVA8Ey0Rwuy34S_lZV5yLHz5mDks5qmFcvgaJpZM4JzSqF>.